### PR TITLE
docs(CONTRIBUTING): Add CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by pcgilday**
## Proposed changes

The commit format for this repo was a gotcha for me when I opened https://github.com/michael-ciniawsky/postcss-load-config/pull/30, so I figured I'd add this. The other rules are from the checklist used by the template.

## Types of changes

What types of changes does your code introduce
_Put an `x` in the boxes that apply_

- [ ] Bug (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature which changes existing functionality)
- [x] Documentation

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes are merged and published in downstream modules

### CODE REVIEW

**Comment by GitCop** [comment]
:warning: Pull Request - Issues :warning:

 * Commit: 3d169bfd770e5f25e1cb49e0b291e64cc694cffa
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**



---
This message was auto-generated by https://gitcop.com

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- Commit summaries must be in the following format `(<type>)<scope>: <description>`.
+  - type - `feat`, `fix`, `docs`, `style`, `refactor`, `perf`, `test`, `chore`, `revert`
+  - scope - file affected, or general scope or commit
```
Comment (SpainTrain) [inline CONTRIBUTING.md]: Think you mean "scope of commit"?

Comment (michael-ciniawsky) [inline CONTRIBUTING.md]: `(<type>)<scope>: <description>` => `<type>(<scope>): description>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - description - Your commit summary
+
+- Write tests that cover your changes
+
+- Make sure `npm run lint` and `npm test` both pass before submitting PR.
+
+- Add docs if appropriate
```
Comment (michael-ciniawsky) [inline CONTRIBUTING.md]: - Add yourself as Contributor in README.md

[Github API](https://api.github.com/user/)

```html
<td align="center">
  <img width="150 height="150"
    src="https://avatars.githubusercontent.com/u/{{ id }}?v=3&s=150">
  <br />
  <a href="https://github.com/{{ login }}">Name</a>
</td>
```

But I can add that later myself aswell :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (pcgilday) [inline CONTRIBUTING.md]: 👍 Both are good catches, thanks.

File: CONTRIBUTING.md (hunk 2 above)
Comment (michael-ciniawsky) [inline CONTRIBUTING.md]: 👍

**Pull request merged**